code/Entities/AccessTokenEntity.php

```diff
@@ -65,7 +64,7 @@ public function getExpiryDateTime()
 
     public function getUserIdentifier()
     {
-        return $this->MemberID;
+        return $this->Client()->MemberID;
```

Comment:
this isnt required, I abandoned the member object / login for oauth. But this wasn't connected to the client entity anyway so cool to leave it in the POC.